src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -11,12 +11,10 @@
     <ProjectGuid>{8772BC91-7B55-49B9-94FA-4B1BE5BEAB55}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableWinRT Condition="'$(TargetGroup)' == 'netcore50'">true</EnableWinRT>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
```

Comment:
That was per Eric's feedback.  

> ericstj added a note 7 days ago
> Same as previous. Also, we aren't packaging the net46 facade so we should remove it from the src/.builds file.
> 

I can add back the "IsPartialFacadeAssembly" property, that was a bit aggressive; but I'll leave the TargetGroup for net46 out of the builds file.